pkg/resources/server_test.go

```diff
@@ -357,7 +377,13 @@ var _ = Describe("resourceServer tests", func() {
 	})
 	Context("ListAndWatch", func() {
 		It("Get devices of plugin and change device status", func() {
-			obj, err := newResourceServer(&types.UserConfig{RdmaHcaMax: 100, ResourceName: "fake"}, true, "fake", "fake")
+			fs := utils.FakeFilesystem{
```

Comment:
Why do you need a fake filesystem now ? 
was it needed before and it was a bug that fake filesystem was not used ?

Reply:
The previous behavior allows creating the resources even if the devices doesn't exist or there is empty list of devices, this test was depending on the second case.